src/components/shareable-url-modal/shareable-url-modal.js

```diff
@@ -4,35 +4,37 @@ import { connect } from 'react-redux';
 import classnames from 'classnames';
 import { toggleShareableUrlModal } from '../../actions';
 import { fetchPackageCompatibilities } from '../../utils';
+import { saveLocalStorage, loadLocalStorage } from '../../store/helpers';
 import {
-  hostingPlatform,
+  hostingPlatforms,
   inputKeyToStateKeyMap,
+  localStorageSharableUrl,
   KEDRO_VIZ_PUBLISH_DOCS_URL,
-  KEDRO_VIZ_PREVIEW_DATASETS_DOCS_URL,
   KEDRO_VIZ_PUBLISH_AWS_DOCS_URL,
   KEDRO_VIZ_PUBLISH_AZURE_DOCS_URL,
   KEDRO_VIZ_PUBLISH_GCP_DOCS_URL,
   PACKAGE_FSSPEC,
 } from '../../config';
 
 import Button from '../ui/button';
-import CopyIcon from '../icons/copy';
+import InfoIcon from '../icons/info';
 import Dropdown from '../ui/dropdown';
 import IconButton from '../ui/icon-button';
 import Input from '../ui/input';
 import LoadingIcon from '../icons/loading';
 import Modal from '../ui/modal';
 import MenuOption from '../ui/menu-option';
-import Tooltip from '../ui/tooltip';
+import Toggle from '../ui/toggle';
 
 import './shareable-url-modal.scss';
+import UrlBox from './url-box/url-box';
 
 const modalMessages = (status, info = '') => {
   const messages = {
     failure: 'Something went wrong. Please try again later.',
     loading: 'Shooting your files through space. Sit tight...',
     success:
-      'The current version of Kedro-Viz has been published and hosted via the link below.',
+      'The deployment has been successful and Kedro-Viz is hosted via the link below..',
     incompatible: `Publishing Kedro-Viz is only supported with fsspec>=2023.9.0. You are currently on version ${info}.\n\nPlease upgrade fsspec to a supported version and ensure you're using Kedro 0.18.2 or above.`,
```

Comment:
Still think these hard coded messages should be in config.js and not here.

Reply:
done, moved it to config